client/src/styles/home.css

```diff
@@ -0,0 +1,33 @@
+.home__banner__wrapper {
+  margin: 30px;
+  top: 20%;
+  position: absolute;
+}
+
+.home__banner__title1 {
+  color: var(--purple);
+}
+.home__banner__title2 {
+  color: var(--beige);
+  background-color: var(--purple);
+  width: fit-content;
```

Comment:
never heard of fit-content and it's not doing anything in my browser.

Reply:
he probably meant either min-content or max-content I imagine

Reply:
fit content just fits the block size to the text-length